modules/appeals_api/README.yml

```diff
@@ -64,7 +64,7 @@ paths:
       summary: Retrieve appeals status for the Veteran with the supplied ssn
       operationId: getAppealStatus
       security:
-        - api_key: []
+        - apikey: []
```

Comment:
Did the name change? Or was this bad documentation?

Reply:
So it's being changed to align with the security schema. The other PR that's tied to this is generating curls from swagger, which requires properly formatted swagger.